src/http/params.cr

```diff
@@ -69,16 +69,25 @@ module HTTP
       end
     end
 
-    # Creates an `HTTP::Params` instance from the key-value
-    # pairs of the given *hash*.
-    def self.from_hash(hash : Hash)
+    # Returns the given key value pairs as a
+    # url-encoded HTTP form/query.
+    def self.encode(hash : Hash)
```

Comment:
Maybe we should restrict to `Hash(String, _)`. I know it isn't part of the initial PR intent.

If we want to work with any type of hash we can merge the implementation.
If we want to support only String key hashes we could restrict here and catch the compile time error sooner.

`Hash(String, _)` sounds fine to me.